openhtf/core/test_record.py

```diff
@@ -71,16 +71,15 @@ def add_outcome_details(self, code, description=''):
 class PhaseRecord(  # pylint: disable=no-init
     mutablerecords.Record(
         'PhaseRecord', ['name', 'codeinfo'],
-        {'measurements': None,
-         'start_time_millis': None, 'end_time_millis': None,
-         'attachments': dict, 'result': None})):
+        {'start_time_millis': None, 'end_time_millis': None,
+         'measurements': dict, 'attachments': dict, 'result': None})):
```

Comment:
hm, why is this changing? seems unrelated?  measurements now defaults to a dict?